charts/kong/example-values/README.md

```diff
@@ -52,5 +52,15 @@ common Kong deployment scenarios on Kubernetes.
   restarts them, so starting control planes second, but around the same time
   will usually work), but is the smoothest option.
 
+* [minimal-kong-scd-controller.yaml](minimal-kong-scd-controller.yaml) and
+  [minimal-kong-scd-gateway.yaml](minimal-kong-scd-gateway.yaml) install a
```

Comment:
"SCD" means nothing outside of KIC dev. I'd say that this is an internal feature codename more than anything.

suggestion: replace "SCD" with some variation of "service discovery".

BTW I'm inclined to think that "gateway discovery" might be more accurate/helpful than "service discovery". Naming things is hard 🤔

Reply:
I like "gateway discovery" 👍

Reply:
That should be fine although I agree that "service discovery" still sounds quite wide and could mean many other things while narrowing it down to "gateway discovery" gives a better grasp of what the feature really does (I've just cross-checked that with a DevOps engineer 😅).

Reply:
To echo my other comment, short abstract names for deployment strategies are always a bit iffy, similar to how "hybrid mode" doesn't really capture what hybrid mode actually entails at a technical level.

I think we largely want the naming to be concise and consistent, and then have a longer docs site article explaining the technical details (controller is de-coupled from Kong instances, uses the admin Service to determine where it needs to send config, which has X/Y/Z benefits over existing strategies, etc.) in depth, so that the name serves less a descriptive purpose than it does a recognizable short name associated with a larger set of concepts.

It's at least much less confusing than the old "Kong for Kubernetes"/"Kong for Kubernetes With Kong Enterprise"/etc. mishmash of near-identical names referring to subtle (but important) distinctions in deployment strategy :laughing: